infra/mybatis3/projectName-domain/src/main/resources/META-INF/mybatis/mybatis-config.xml

```diff
@@ -9,10 +9,11 @@
         <setting name="mapUnderscoreToCamelCase" value="true" />
         <setting name="lazyLoadingEnabled" value="true" />
         <setting name="aggressiveLazyLoading" value="false" />
+        <setting name="defaultFetchSize" value="100" />
 <!--
         <setting name="defaultExecutorType" value="REUSE" />
         <setting name="jdbcTypeForNull" value="NULL" />
-        <setting name="proxyFactory" value="JAVASSIST" />
+        <setting name="proxyFactory" value="CGLIB" />
```

Comment:
- [ ] Is this change really needed? In my understanding, Java Assist should be usually used as the proxy factory. This line seems that we are recommending cglib.